lib/template-mixins.js

```diff
@@ -524,12 +536,15 @@ module.exports = function (options, deprecated) {
                         throw new Error('Could not find field: ' + key);
                     }
                 }
+                if (this.disableRender) {
+                    return null;
+                }
                 if (this.html) {
                     return this.html;
                 }
                 var mixin = this.mixin || 'input-text';
                 if (mixin && res.locals[mixin] && typeof res.locals[mixin] === 'function') {
-                    return res.locals[mixin]().call(Object.assign({}, res.locals, this), this.key);
+                    return res.locals[mixin]().call(Object.assign({}, res.locals), this.key);
```

Comment:
`this` (field in `res.locals.fields`) extends the actual field config here, so if `options` is defined on the field like in a radio group then further up in the actual mixin method, this.options has been overwritten - `this` shouldn't have been exposed in the first case, and it only wasn't a bug before as we were limiting the keys added to res.locals.fields